### DISCUSSION TITLE
Overlap with Harmonize Mobility ISO Standard

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#10] [Data Specification]
This CDS scope and purpose seems overlapping with the [non-open] ISO spec discussed here: https://harmonizemobility.com/wp-content/uploads/2020/03/Sidewalk-Kerb-Standard-invitation-Purpose-Justification-Q1-2020.pdf

Is there a larger conversation to be had (or that has already happened) to discuss (a) distinct roles for this spec vs ISO one? and (b) use of open standards vs [arguably] closed ones?

Start of Answer:
The OMF and Steering Committee has been in contact and talked to people leading this ISO initiative. But we believe there is not directly overlap of short term scope (ISO standard a long way off) or timeline yet. It's more a definition of the physical curb, not data exchange at curbs, except in regards to security around that exchange. After v1.0 of CDS I think there is a possibility of closer alignment with a breaking change in 2.0.

**Comment by jacobmalleau** [comment]
IBI Group has been involved in this work as it relates to curbside operations and facilitating access to sidewalks for AVs. As we understand it they are focusing on operating principles (a digital representation of curbside regulations should exist to facilitate AV access to sidewalks) rather than formal data standards and specifications. A draft document on their work to date should be made public later this fall.
Because the Harmonize ISO work is not focused on defining a data standard or spec for curbside operations, I don't see a conflict or need for closer collaboration with that effort at this time.

It is also worth noting the HM is undergoing a very open discussion with a variety of stakeholders to help define this spec, so I would not say it is a closed standard per se.
